### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nishinji**
RAG also said it should be changed.
I was able to get it to work with jekyll new instead of bundle install.

### CODE REVIEW

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]
just bundle install, sudo is required on linux, dunno about windows

**Comment by nishinji** [comment]
Corrected
I believe it was only needed the first time

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ bower_components
 _site
 .jekyll-cache
 .jekyll-metadata
+/.vs
```
Comment (EXtremeExploit) [inline .gitignore:6]: :thinking: why?. also `.vs` is fine, else it becomes a non-relative path

**Comment by IllusionMan1212** [comment]
please resolve the conflicts

**Pull request merged**